src/components/ADempiere/ModalDialog/index.vue

```diff
@@ -246,6 +256,19 @@ export default defineComponent({
     }
 
     const doneButton = () => {
+      store.dispatch('setModalDialog', {
+        ...storedModalDialog.value,
+        isDisabledDone() {
+          return true
+        }
```

Comment:
https://github.com/solop-develop/frontend-core/blob/feature/legacy-support/src/store/modules/ADempiere/dictionary/window/actions.js#L273-L279

Here you are overwriting the original callback, so if you want to validate with the original you can't because you lose it.